metadata.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "puppetlabs-yumrepo_core",
+  "version": "1.0.0",
+  "author": "Puppet Labs",
+  "summary": "Manage client yum repo configurations by parsing yum INI configuration files.",
+  "license": "Apache-2.0",
+  "source": "https://github.com/puppetlabs/puppetlabs-yumrepo_core",
+  "project_page": "https://puppet.com/docs/puppet/latest/types/yumrepo.html",
+  "issues_url": "https://tickets.puppetlabs.com/projects/MODULES",
+  "dependencies": [
+
+  ],
+  "operatingsystem_support": [
```

Comment:
I'm not 100% sure about this list.

Reply:
Should includes versions 5 and 6 of RHEL and CentOS (and oracle/scientific?).

I'm unsure about Fedoras, since they've switched to DNF. I guess the repo files are pretty similar? We should confirm that.

Reply:
It appears recent fedoras still use [yumrepo configs](https://www.rootusers.com/25-useful-dnf-command-examples-for-package-management-in-linux/). It looks like you can drop a file in `/etc/yum.repos.d` or use `dnf config-manager`, which does the same. It also mentions configuring them in `dnf.conf`, but I imagine if you did that, puppet wouldn't know about it.

Reply:
In that case, the currently supported fedora versions are 26, 27, and 28, with 29 coming out very shortly and us probably dropping 26 very soon